### ISSUE TITLE
VPAT 2.1.1b, Keyboard access changes with Voiceover turned on

### ISSUE THREAD

**Issue by david-iterators** [CityOfBoston/Iterators#16]
Screengrab/videos (insert file names here)
4, Keyboard, Safari, Voiceover on, menu disappears, edit-shorten.mp4

Link
https://permit-finder.digital-staging.boston.gov

Environment
(Device > OS > Browser)
Computer > macOS 10.14.6 > Safari 12.1.2

Brief Description
Interactive elements are not accessible through the keyboard, and the behavior changes when VoiceOver is activated.

Steps to reproduce
1.  0:00 Look at “4, Keyboard, Safari, Voiceover on, menu disappears, edit-shorten.mp4” 
2.  0:33  Shift-Tab so that the menu is activated by going backwards (previously reported).  Notice that all submenu items are accessible.
3.  0:47  Activate voiceover
4.  0:51 Shift-Tab to activate the menu --- this works at this time.
5.  1:15  Continue tabbing off of the submenu.  Notice that some of the items have now scrolled off screen, such as the Public Notices and the Search icon
6.  1:17 Table to Public Notices, which is now off-screen, the expanded menu disappears, but does not collapse back.
7.  1:21  Shift-Tab to go backwards to the submenu.  Notice that the submenu is still active, but is not visible.

Desired Result
Keyboard use is compatible with VoiceOver

Actual Result
The submenu disappears under some circumstances when VoiceOver is activated.
[4, Keyboard, Safari, Voiceover on, menu disappears.zip](https://github.com/CityOfBoston/Iterators/files/3467655/4.Keyboard.Safari.Voiceover.on.menu.disappears.zip)

**Comment by paylakatel** [comment]
This also seems to be a patterns/D8 issue with the menu/header - not permit finder, we're just pulling in this source code. I think it should be addressed when the entire patterns library is looked at.
